src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Instead of using the literal value 20, define a constant to explain its meaning. For example, 'const MID_TERM_DISCOUNT = 20;'.